wagtail/wagtailadmin/utils.py

```diff
@@ -62,6 +64,64 @@ def users_with_page_permission(page, permission_type, include_superusers=True):
     return User.objects.filter(is_active=True).filter(q).distinct()
 
 
+def permission_denied(request):
+    "Return a standard 'permission denied' response"
```

Comment:
PEP 257 recommends using tripple-quoted strings even for single line docstrings: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings